package.json

```diff
@@ -16,10 +16,12 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.2",
+    "chai-as-promised": "^7.1.1",
```

Comment:
Because this is only used in your tests, you can move this dependency into `devDependencies` and re-run `npm install`.

Reply:
Cool! Will do!